eox_nelp/course_experience/api/v1/views.py

```diff
@@ -228,13 +228,15 @@ class LikeDislikeUnitExperienceView(UnitExperienceView):
                     "author": {
                         "data": {
                             "type": "User",
-                            "id": "7"
+                            "id": "7",
+                            "attributes": {}
```

Comment:
why empty if you already added  USER_EXTRA_ATTRIBUTES.

Reply:
Because it could be variable ?